### PR TITLE
(gnucash) Migrated to AU + Modernization [AU gnucash](unreject)

### LINKED ISSUE #296: [coreteampackages] gnucash v2.6.14 - Action Required - Failed Verification Testing
'gnucash' is Submitted.
Current status = Waiting for Maintainer to take corrective action

chocolatey-ops left the following comment(s):

gnucash has failed automated testing.
This is not the only check that is performed so check the package page to
ensure a 'Ready' status.
Please visit https://gist.github.com/3cfd638209e7c636f671420012f37765 for
details.
The package status will be changed and will be waiting on your next actions.

Package Url: https://chocolatey.org/packages/gnucash/2.6.14

### PR DESCRIPTION
**Pull request by AdmiringWorm**
@gep13 version [2.6.14](https://chocolatey.org/packages/gnucash/2.6.14) needs to be unrejected

fixes #296

### CODE REVIEW

**Comment by majkinetor** [comment]
LGTM, I will leave @gep13 to do this since it requires unrejection.

**Comment by majkinetor** [comment]
BTW, I used github instead sourceforge:

```powershell
import-module au

$releases = 'https://github.com/Gnucash/gnucash/releases'

function global:au_SearchReplace {
   @{
        ".\tools\chocolateyInstall.ps1" = @{
            "(?i)(^\s*url\s*=\s*)('.*')"          = "`$1'$($Latest.URL32)'"
            "(?i)(^\s*checksum\s*=\s*)('.*')"     = "`$1'$($Latest.Checksum32)'"
            "(?i)(^\s*packageName\s*=\s*)('.*')"  = "`$1'$($Latest.PackageName)'"
            "(?i)(^\s*softwareName\s*=\s*)('.*')" = "`$1'$($Latest.PackageName)*'"
            "(?i)(^\s*fileType\s*=\s*)('.*')"     = "`$1'$($Latest.FileType)'"
        }
    }
}

function global:au_GetLatest {
    $download_page = Invoke-WebRequest -Uri $releases

    $re  = '^/.+\.exe$'
    $url = $download_page.links | ? href -match $re | select -First 1 -expand href
    @{
        Version = $url -split '-' | select -Last 1 -Skip 1
        URL32   = 'https://github.com' + $url
    }
}

update -ChecksumFor 32
```

**Comment by AdmiringWorm** [comment]
@majkinetor ah yes, that would definitely be better.
Unfortunately I'm not in a position to update the script, would you mind doing it?
Otherwise it will have to wait until tomorrow

**Comment by majkinetor** [comment]
I have teirribly slow connection ATM and this is big file. So I am stuck too.

**Comment by majkinetor** [comment]
One note tho, I find that above `re` must be used as there are invalid links in the release body and links to other domains. So this one should target only github stuff.

**Comment by AdmiringWorm** [comment]
ah, forgot I could edit through github. can't test it until tomorrow though

File: automatic/gnucash/update.ps1 [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    ".\tools\chocolateyInstall.ps1" = @{
+      "(?i)(^\s*url\s*=\s*)('.*')"            = "`$1'$($Latest.URL32)'"
+      "(?i)(^\s*checksum\s*=\s*)('.*')"       = "`$1'$($Latest.Checksum32)'"
+      "(?i)(^\s*checksumType\s*=\s*)('.*')"   = "`$1'$($Latest.ChecksumType32)'"
+    }
+  }
+}
+
+function global:au_GetLatest {
+  $download_page = Invoke-WebRequest -UseBasicParsing -Uri $releases
```
Comment (majkinetor) [inline automatic/gnucash/update.ps1]: UBP on github ?

Comment (majkinetor) [inline automatic/gnucash/update.ps1]: Hm... I checked some other stuff and now they work too !

Comment (majkinetor) [inline automatic/gnucash/update.ps1]: How can you not like when long standing problems solve on its own...

Comment (AdmiringWorm) [inline automatic/gnucash/update.ps1]: seems like the problem isn't yet gone, I just tried to run it on an appveyor build.
and it has been hanging for the last 2 minutes.

Comment (majkinetor) [inline automatic/gnucash/update.ps1]: Meh... 
By default AU should terminate it after 20 mins.

**Pull request merged**